src/main/java/dev/breeze/settlements/entities/cats/VillagerCat.java

```diff
@@ -281,4 +299,14 @@ public void tameByVillager(@Nonnull BaseVillager villager, CatVariant variant) {
         this.setHealth(this.getMaxHealth());
     }
 
+    public void onOwnerSensorTick() {
+        // Check if we've already registered the behaviors
+        if (this.behaviorsRegisteredSuccessfully) {
+            return;
+        }
+
+        LogUtil.info("Owner detected, refreshing cat brain goals");
```

Comment:
Maybe make this a debug message instead?

Reply:
Yea I had the same idea, but I felt like refactoring (and adding) an entire debug util class might be better in a separate PR, so I left this as is.

Reply:
Yea I had the same idea, but I felt like refactoring (and adding) an entire debug util class might be better in a separate PR, so I left this as is.

Reply:
Ah sounds good!